BruTile/Wmts/WmtsCapabilitiesParser.cs

```diff
@@ -89,6 +89,12 @@ private static List<HttpTileSource> GetLayers(Capabilities capabilities, List<Wm
 
                 foreach (var style in layer.Style)
                 {
+                    var styleName = "default";
```

Comment:
To make it a bit more compact I think I would prefer not to use a variable and to use `style.Identifier.Value ?? "default"` on the line where the variable is used.